# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/mips32/Assembler-mips32.h
# Commit: 8e95ddbb6b20
# Full Hash: 8e95ddbb6b20a3532d92e6bcf106f75d5565cf52
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2020-11-03 21:39:33
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Part 1: Reserve two slots after stack arguments for the future tls preservation. r=lth
#   
#   We are going to remove Frame::tls and support trampolines for indirect calls, so we need to get rid of using Frame::tls.
#   In this and the followup patches I will iteratively remove all dependencies of Frame::tls and remove it eventually.
#   
# ==============================================================================

diff -r f8862431f979 -r 8e95ddbb6b20 js/src/jit/mips32/Assembler-mips32.h
--- a/js/src/jit/mips32/Assembler-mips32.h	Tue Nov 03 14:32:33 2020 +0000
+++ b/js/src/jit/mips32/Assembler-mips32.h	Tue Nov 03 10:18:03 2020 +0000
@@ -43,6 +43,8 @@
 
     return usedArgSlots_ * sizeof(intptr_t);
   }
+
+  void increaseStackOffset(uint32_t bytes) { MOZ_CRASH("NYI"); }
 };
 
 // These registers may be volatile or nonvolatile.